pkg/generated/restapi/operations/timestamp_server_api.go

```diff
@@ -350,6 +350,6 @@ func (o *TimestampServerAPI) AddMiddlewareFor(method, path string, builder middl
 	}
 	o.Init()
 	if h, ok := o.handlers[um][path]; ok {
-		o.handlers[method][path] = builder(h)
+		o.handlers[um][path] = builder(h)
```

Comment:
this is an unexpected change, i wonder if we don't have the swagger tool pinned?

Reply:
Should be - https://github.com/sigstore/timestamp-authority/blob/main/hack/tools/go.mod#L6

Reply:
Did you run `make` to generate this?

Reply:
My guess is it's from updating https://github.com/sigstore/timestamp-authority/pull/386, and we don't check that no files would be changed.